data/geo_groups.json

```diff
@@ -35,6 +35,49 @@
       ]
     }
   },
+  "NV": {
```

Comment:
none of these overlap except that there are 2 that serve Clark county - HELP serves Southern Rural Clark County and NRHA serves Northern Rural Clark County. I added clark county to both geogroups but that may not be the right approach. I suspect this is an edge case because the majority of people who live in clark county are las vegas, north las vegas, or henderson, which are listed specifically under cities. it might be better in that case to remove clark county from the list to avoid showing incorrect information -- what do people think?

Reply:
The semantics of having multiple authority types in a group is "AND" -- i.e. "you must be in one of these cities AND in one of these counties". So `help-of-southern-nevada` is overly restrictive: it will only match if you're within Vegas or Henderson, and the county part is redundant.

`nevada-rural-housing` will only match if you're in North Las Vegas (and thus automatically in Henderson County). Carson City is a county-equivalent, so it will never match because there isn't an object under `counties` with `"county": "Carson City"`. (The `county` key in the objects under `cities` is meaningless; it's never read. We could stand to have a tighter schema on authorities.json, it looks like.)

I'm pretty sure the current semantics of geo groups don't support these cases fully. Do we need the ability to do negation here? Like "Henderson County AND NOT [these cities]"? (I think there's a case in Colorado that would benefit from this too, fwiw.)

Reply:
BTW, you may be wondering: why is it AND semantics, that seems nonsensical where cities and counties are concerned?

I was mostly thinking about combining either a cities or counties restriction with a utilities restriction, like "customers of XYZ Utility who live in City of ABC" or whatever.  I also realized you could simulate OR semantics by having two different geo groups (one for each side of the OR) and copying the incentive, with each copy having one of the two geo groups.

I didn't account for NOT semantics in any way, which I think is called for here.

Reply:
ah, good to know that this is AND -- I think there's an okay solution that we can get to here for `help-of-southern-nevada` and `nevada-rural-housing-north`. we could:
1.  update `help-of-southern-nevada` to only include the cities of LV and Henderson and remove clark county
2.  split `nevada-rural-housing-north` into 2 groups/incentives to create an OR that applies to city of north las vegas OR any of the counties

this should cover the whole state except for rural clark county, because clark county won't be included in any of these, otherwise we'd need to support the NOT functionality. I actually think that's okay because northern clark county is served by a different agency than southern clark county (but the website doesn't specify what north and south refer to specifically). since the main population centers in clark county are las vegas, north las vegas, and henderson, that should cover most people, and we would just have to exclude rural clark county for now (which seems acceptable?). alternatively we could just combine all of these into 1 state-wide program and leave it up to users to figure out which one to apply to, but the problem with that is that each of the 4 programs has different low income thresholds. what do you think?

something like:
```
"nv-group-help-of-southern-nevada": {
  "cities": [
    "nv-city-of-las-vegas",
    "nv-city-of-henderson"
  ]
},
"nv-group-nevada-rural-housing-north-lv": {
  "cities": [
    "nv-city-of-north-las-vegas"
  ]
},
"nv-group-nevada-rural-housing": {
  "counties": [
    "nv-churchill-county",
    "nv-douglas-county",
    "nv-lyon-county",
    "nv-storey-county",
    "nv-carson-city"
  ]
},
```

Reply:
updated in the[ most recent commit](https://github.com/rewiringamerica/api.rewiringamerica.org/pull/345/commits/af002e3cd0f164c1229651c9f06ad65df669a484) and tested with zip codes in las vegas, north las vegas, henderson, and carson city!

Reply:
OK, this seems fine -- I also can't find anywhere that defines what "northern rural Clark County" is, so I'm OK with punting on it.

Also, why would they include North Las Vegas in "Nevada Rural Housing"? It's very much not rural!